### ISSUE TITLE
Loading animation

### ISSUE THREAD

**Issue by maxvisser** [teamforus/forus#1111]
## Context/goal: 
It would be nice to have a loading animation when the frontend is waiting on a request from the backend. Some pages, actions will have 3+ seconds to load. User will try to click multiple times or do unintended actions while waiting if he is not informed the action is going through.

@danrminds crm has a nice loading animation, would be nice to implement this on dashboard/webshop aswell.

**Comment by maxvisser** [comment]
nice issue count like the simitry 1/1

**Comment by jamalv** [comment]
https://github.com/teamforus/forus/issues/1179

**Comment by maxvisser** [comment]
done merged to develop
